src/ai/backend/manager/models/minilang/ordering.py

```diff
@@ -56,6 +56,10 @@ def _get_col(self, col_name: str) -> sa.Column:
                     case JSONFieldItem(_col, _key):
                         _column = get_col_from_table(self._sa_table, _col)
                         matched_col = _column.op("->>")(_key)
+                    case JSONArrayFieldItem(_col_name, _conditions, _key_name):
+                        # TODO: Implement this.
+                        pass
+                        # ...
```

Comment:
Was it handled in a follow-up PR?

Reply:
This code requires discussion before going ahead with work.
I created a Teams thread, but I haven’t received any responses yet.